### DISCUSSION TITLE
[Feature Request] Allow adding attributes to Filter Slidedown Menu

### DISCUSSION THREAD

**Discussion by lrljoe** [rappasoft/laravel-livewire-tables#1474] [Feature Requests]

**Comment by tychoengberinkDIJ** [comment]
Please add this to the package

**Comment by tychoengberinkDIJ** [comment]
Additionally, implement this also for the sorting pills and filter pills to allow customization of their colors.

**Comment by lrljoe** [comment]
Filter Slidedown attributes should be available later this month.

setFilterPopoverAttributes is already available.

Styling of the filter pills is somewhat possible already using existing methods, but I will look to add the attribute methods in

**Reply by lrljoe** [reply]
@tychoengberinkDIJ  - Which section(s) of the Filter SlideDown are most critical for you in terms of attributes?  

As there are several elements that make up the slidedown, so want to make sure I prioritise what you need most urgently.

**Reply by tychoengberinkDIJ** [reply]
When implementing I need to update the following files to use the right colors. Otherwise its indigo. I would be nice to have attribute methods to change it. Now i need to manually update the vendor files to make the files compatible when there is an update.

Maybe implement a global color setting if possible?

![afbeelding](https://github.com/user-attachments/assets/4bf518fb-b33f-48fe-a5fb-455396a7277c)

**Reply by lrljoe** [reply]
So a global color is trickier than you'd think, when you take into account how many sites are already using the package, plus the Tailwind vs BS4 vs BS5 support

I'll have a think though!  Welcome any suggestions
